tests/config.h

```diff
@@ -10,7 +10,7 @@
 #ifndef __CONFIG_H
 #define __CONFIG_H
 
-#include "clockcache.h"
+#include "clockcache.h" // Nested #include's task.h and related tokens.
```

Comment:
Why not include "task.h" directly?  Also, clockcache.h should not include task.h.

Reply:
Investigate ... source of `.h` comment and clean up as best as you can.

Reply:
@gapisback were you going to address this before merging?

Reply:
@rosenhouse - I did not try to 'resolve' these nested `.h` inclusions. This comment was needlessly drawing attention and I took it out.

(A separate issue will need to be logged to clean up these nested includes. I'm not following up on that.)